block-storage-attacher/tests/e2e/e2e-tests/e2e_suite_test.go

```diff
@@ -14,7 +14,7 @@ package e2e
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e/framework"
+	"github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/framework"
```

Comment:
This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.